### ISSUE TITLE
Chain Lightning cast at high level is being interpreted as a second roll

### ISSUE THREAD

**Issue by trey-kirk-sp** [mr-ice/maptool-macros#61]
Cast Chain Lightning at 7th level, get:

[{   "name": "Chain Lightning",   "diceSize": 8,   "diceRolled": 10,   "expressionTypes": "Damage",   "bonus": "0",   "totalRolls": 2,   "damageTypes": "Lightning",   "output": "Chain Lightning Damage Roll: 10d8 + 0 = 20 Lightning",   "rolls":   [     45,     20   ],   "roll": 20,   "rollString": "10d8 + 0",   "total": 20,   "totals":   [     45,     20   ],   "outputs":   [     "Chain Lightning Damage Roll: 10d8 + 0 = 45 Lightning",     "Chain Lightning Damage Roll: 10d8 + 0 = 20 Lightning"   ],   "allTotal": 65 }]
--
Note: totalRolls == 2, should still be 1. My hunch is the logic to identify Magic Missile is misidentifying here.

**Comment by trey-kirk-sp** [comment]
As expected, the modifier for Chain Lightning is the same version as Magic Missile, which is devoid anything useful:
```
 {
      "atHigherLevels":       {
        "additionalAttacks": [],
        "additionalTargets": [],
        "areaOfEffect": [],
        "creatures": [],
        "duration": [],
        "higherLevelDefinitions": [],
        "points": [],
        "scaleType": "spellscale",
        "special": []
      },
```
So need to revert the hook for Magic Missile to just check if its magic missile and otherwise ignore empty modifiers like this.

**Comment by trey-kirk-sp** [comment]
Fixed in hash d65618e9
